src/report/generate_trustymail_report.py

```diff
@@ -1098,9 +1098,11 @@ def process_record(domain, record, policy_published):
             if "reason" in policy_evaluated:
                 reason = policy_evaluated["reason"]
                 if isinstance(reason, list):
-                    x["Override Reason"] = " ".join([rsn["type"] for rsn in reason])
+                    x["Override Reason"] = " ".join(
+                        [str(rsn["type"]) for rsn in reason]
+                    )
```

Comment:
I think we can use a generator here instead of explicitly creating a list.

```suggestion
                    x["Override Reason"] = " ".join(
                        str(rsn["type"]) for rsn in reason
                    )
```